src/caselaw/index.html

```diff
@@ -82,7 +82,7 @@
 	<body>
 		<cap-nav></cap-nav>
 		<main id="main">
-			<cap-content-router></cap-content-router>
+			<cap-content-router class="c-tall"></cap-content-router>
```

Comment:
This works. I might rename this `u-min-height` to convey that it's a utility class, rather than a component class, and to indicate what specific property is being applied. A similar utility class might be applying a max width to a container— which might be called, for example, say `u-container-max-width` or just `u-max-width`

Reply:
Sometimes if I don't know what to name a utility class, I take a peak at the Tailwind documentation, and pilfer it for inspiration: https://tailwindcss.com/docs/installation

Reply:
I am sure that soon, I will come to understand the distinctions between these types of classes :-) Thanks for the correction!